### PR TITLE
knife: Improve messaging for installing official plugins into DK / Workstationn

### PR DESCRIPTION
**Pull request by vsingh-msys**
Signed-off-by: Vivek Singh <vivek.singh@msystechnologies.com>

## Description

https://github.com/chef/chef/blob/f6d445c28467b5ea5741d05aed2bd1f657c4a1dd/lib/chef/knife.rb#L237
- knife plugins list variable `vsphere`, `acl` , `opc`, `tidy` are missing.
- Add these as well and sorted the list.

## Related Issue

https://github.com/chef/chef/issues/8898

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Comment by vsingh-msys** [comment]
https://github.com/chef/chef/blob/f6d445c28467b5ea5741d05aed2bd1f657c4a1dd/lib/chef/knife.rb#L276
@tas50 & @lamont-granquist we can also think of updating the text `ChefDk`?

**Comment by tas50** [comment]
These were left out intentionally as they're already in DK / Workstation. The only thing we need to mention here is the stuff you'd need to install by hand.

**Comment by vsingh-msys** [comment]
@tas50 Okay got it thanks, seems a couple of plugins we can remove `windows`, `google`, `azure` and `ec2` these are already shipped into chefdk/workstation.

ChefDk: https://github.com/chef/chef-dk/blob/24a74c1f745812852db2396e32fa715d2ff100ea/Gemfile#L72-L79

Workstation: https://github.com/chef/chef-workstation/blob/master/components/gems/Gemfile#L79-L88

**Comment by vsingh-msys** [comment]
Removed `windows, google, azure & ec2` knife plugins these are already shipped into chefdk 4+.

File: lib/chef/knife.rb [hunk 1]
```diff
@@ -234,7 +234,7 @@ def self.load_deps
       dependency_loaders.each(&:call)
     end
 
-    OFFICIAL_PLUGINS = %w{ec2 rackspace windows openstack azure google linode push vcenter lpar}.freeze
+    OFFICIAL_PLUGINS = %w{linode lpar openstack push rackspace vcenter}.freeze
```
Comment (tas50) [inline lib/chef/knife.rb]: linode is end of life so that should be removed. Push is part of Workstation and vcenter is in Workstation now.

**Comment by tas50** [comment]
The line below that mentions ChefDK should be changed to "ChefDK / Chef Workstation"

File: lib/chef/knife.rb (hunk 1 above)
Comment (vsingh-msys) [reply lib/chef/knife.rb]: Okay @tas50 removed `linode`. I think for now we should keep `push` and `vcenter` till ChefDK EOL :)

**Comment by vsingh-msys** [comment]
@tas50 did suggested changes could you please have a look again?
Thanks!

**Comment by vsingh-msys** [comment]
@tas50 I think this PR good to merge?

**Pull request merged**